packages/connect-web/src/connect-transport.ts

```diff
@@ -204,7 +204,15 @@ export function createConnectTransport(
             service,
             method,
             header: demuxedHeader,
-            message: parse(new Uint8Array(await response.arrayBuffer())),
+            message: useBinaryFormat
+              ? method.O.fromBinary(
+                  new Uint8Array(await response.arrayBuffer()),
+                  options.binaryOptions
+                )
+              : method.O.fromJson(
+                  (await response.json()) as JsonValue,
+                  options.jsonOptions
+                ),
```

Comment:
We are still pulling `parse` into the bundle. Would this shave off a couple of bytes?

```suggestion
            message: useBinaryFormat
              ? parse(new Uint8Array(await response.arrayBuffer()))
              : method.O.fromJson(
                  (await response.json()) as JsonValue,
                  options.jsonOptions
                ),
```